src/m1_facilitator/models/ERC20GatewayTokenPair.ts

```diff
@@ -94,4 +95,15 @@ export default class ERC20GatewayTokenPair extends Comparable<ERC20GatewayTokenP
 
     return this.valueToken.localeCompare(other.valueToken, 'en', { sensitivity: 'base' });
   }
+
+  /**
+   * Generates and return global address of given gateway contract address.
+   *
+   * @param gatewayAddress Anchor contract address.
+   *
+   * @returns Gateway global address.
+   */
+  public static getGlobalAddress(gatewayAddress: string): string {
```

Comment:
remove this method, This method should be in Gateway.

Reply:
Done